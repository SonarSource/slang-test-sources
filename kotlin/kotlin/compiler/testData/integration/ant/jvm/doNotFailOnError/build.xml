<project name="Ant Task Test" default="build">
    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>

    <target name="build">
        <javac srcdir="${test.data}" destdir="${temp}" includeantruntime="false" failonerror="false">
            <withKotlin/>
        </javac>

        <kotlinc src="${test.data}" output="${temp}" failonerror="false"/>
    </target>
</project>
