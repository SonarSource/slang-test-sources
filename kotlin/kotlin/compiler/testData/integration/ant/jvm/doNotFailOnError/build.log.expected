OUT:
Buildfile: [TestData]/build.xml

build:
    [javac] Compiling 2 source files to [Temp]
    [javac] Compiling [[TestData]] => [[Temp]]
    [javac] [TestData]/incorrectKotlinCode.kt:4:1: error: parameter name expected
    [javac]
    [javac] ^
    [javac] [TestData]/incorrectKotlinCode.kt:4:1: error: expecting comma or ')'
    [javac]
    [javac] ^
    [javac] [TestData]/incorrectKotlinCode.kt:4:1: error: expecting ')'
    [javac]
    [javac] ^
  [kotlinc] Compiling [[TestData]] => [[Temp]]
  [kotlinc] [TestData]/incorrectKotlinCode.kt:4:1: error: parameter name expected
  [kotlinc]
  [kotlinc] ^
  [kotlinc] [TestData]/incorrectKotlinCode.kt:4:1: error: expecting comma or ')'
  [kotlinc]
  [kotlinc] ^
  [kotlinc] [TestData]/incorrectKotlinCode.kt:4:1: error: expecting ')'
  [kotlinc]
  [kotlinc] ^

BUILD SUCCESSFUL
Total time: [time]

ERR:
    [javac] Compile failed; see the compiler error output for details.

Return code: 0
